Zero_engine.alp

```diff
@@ -8213,6 +8213,11 @@ for (GridConnection GC : GridBatteries) {
 	if (!GC.v_isActive) {
 		GC.f_setActive(false);
 	}
+}
```

Comment:
Waarom zijn dit eigenlijk meerdere loops over de verschillende GC-types, ipv één loop over c_gridConnections?

Reply:
Euhm, omdat (nog) niet alle Gridconnections van te voren op pauze kunnen worden gezet. (Huizen/bedrijven nog niet zeg maar). Echter wil ik dit wel graag, dus zeker een welkome aanpassing, waarna dit inderdaad gewoon zou kunnen loopen over alle GC. (nu ook al, maar dan wordt de loop veel groter dan nodig, dus denk dat luc daarom heeft gekozen om dat eerst niet te doen).

Reply:
Ok, helder. Deze functie f_initializePause wordt maar één keer aangeroepen, toch? Dan is het minder erg als je over alle gridConnections loopt. (vergeleken met een functie die elke tijdstap wordt aangeroepen)
Maar voor nu prima zo!